src/Traits/Http.php

```diff
@@ -253,15 +267,17 @@ protected function prepareMultipartParams(array $params, $inputFileField): array
 
         $inputFile = $params[$inputFileField];
 
-
+        //All files should be provided in an InputFile object
         if (is_resource($inputFile)) {
             throw CouldNotUploadInputFile::resourceShouldBeInputFileEntity($inputFileField);
         }
 
+        //If the user provides a path to a file, attempt to create an InputFile Object automatically for them.
         if (is_string($inputFile)) {
             $params[$inputFileField] = InputFile::create($inputFile);
         }
```

Comment:
But how do you detect if its a file_id string or a file path string?

I thought you said you wanted to allow file_id to still be passed as a string?